source/features/link-to-gihub-io.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "dom-chef";
+import select from "select-dom";
+import elementReady from "element-ready";
+import * as pageDetect from "github-url-detection";
+import LinkExternalIcon from "octicon/link-external.svg";
+
+import features from ".";
+
+function getCurrentUsername(): string {
+	return location.pathname.replace(/(^\/)/g, "").split("/")[0];
+}
+
+function getGithubIoLink(username: string): string {
+	return `${username}.github.io`;
+}
+
+async function repoTabInit(): Promise<void> {
+	const username = getCurrentUsername();
```

Comment:
Same as before use `getRepositoryInfo`, and you can drop the function `getCurrentUsername` in this file.